README.md

```diff
@@ -1,2 +1,27 @@
 # securedrop-client
-SecureDrop Qubes client
+[![CircleCI](https://circleci.com/gh/freedomofpress/securedrop-client.svg?style=svg)](https://circleci.com/gh/freedomofpress/securedrop-client)
+
+Qt-based client for working with SecureDrop submissions on the SecureDrop Qubes Workstation
+
+## Getting Started
+
+Set up a Python 3 virtual environment and set up dependencies:
+
+```
+mkvirtualenv --python=python3 securedropclient
+pip install -r requirements/requirements.txt
+pip install -r requirements/requirements-dev.txt
+```
+
+## Run tests
+
+```
+python -m unittest
```

Comment:
mostly because I generally stick to the stdlib until I Need A Thing in a dependency, but it's a test dep and I do like fixtures so I added `pytest` 😛